index.html

```diff
@@ -0,0 +1,159 @@
+                    <a href="#" class="nav-list">News</a>
+                </li>
+                <button type="button" id="nav-btn">Join Now</button>
+            </ul>
+        </nav>
+    </header>
+    <div id="hamburger">
+        <a href="#" id="close">
+            <i id="close-icon" class="material-icons">close</i>
+        </a>
+        <ul id="hamburger-ul">
+            <li>
+                <a class="link" href="./index.html"> Home</a>
+            </li>
+            <li>
+                <a class="link" href="#"> Program</a>
+            </li>
+            <li>
+                <a class="link" href="./about.html"> About</a>
+            </li>
+            <li>
+                <a class="link" href="#"> Sponsor</a>
+            </li>
+            <li>
+                <a class="link" href="#"> News</a>
+            </li>
+        </ul>
+    </div>
+    <main>
+        <section id="introduction">
+            <h3 class="introduction-greet">"Hello! Data Scientists"</h3>
+            <h1 class="introduction-h1">AI & BIG DATA FOR A SMARTER FUTURE EXHIBITION 2023</h1>
+            <P class="introduction-para">
+                Data literacy is so critical to our education system but also
+                key to humanity. This Exhibition will showcase the next generation
+                enterprise technologies and strategies from the world of Data Science
+                and AI, providing an opportunity to explore and discover the practical
+                and successful implementation of data science and machine learning to 
+                drive your business forward.
+            </P>
+            <div class="introduction-div">
+                <h2 class="intro-div-h2">2023.01.19(THUR) ~ 20(FRI)</h2>
+                <p class="intro-div-para">@ Riverside Drive, Nairobi, Kenya.</p>
+            </div>
+        </section>
+        <section id="program-section">
+            <h2>Main Program</h2>
+            <hr>
+            <div id="programs">
+                <div class="program">
+                    <img src="images/lecture.png" alt="Lecture">
+                    <h4>Lecture</h4>
+                    <p>Listen to the speakers from various countries about securing value through AI & Big Data.</p>
+                </div>
+                <div class="program">
+                    <img src="images/exhibition.png" alt="Exhibition">
+                    <h4>Exhibition</h4>
+                    <p>Appreciate various creations applying successful implementation of data science and AI.</p>
+                </div>
+                <div class="program">
+                    <img id="" src="images/forum.png" alt="Forum">
+                    <h4>Forum</h4>
+                    <p>Have time to share your thoughts and opinions with experts from each topic.</p>
+                </div>
+                <div class="program">
+                    <img src="images/workshop.png" alt="Workshop">
+                    <h4>Workshop</h4>
```

Comment:
Great job so far 👍. I think in order to actualize the design provided by Microverse for this section, you need the right background. You can make use of the background I will provide below 👇 to make the section look similar to the design.

![Prog-bg](https://user-images.githubusercontent.com/73854999/205441845-489ecbcc-e36f-49b8-925f-a87dde70bccf.jpeg)